src/indexer/app.rs

```diff
@@ -0,0 +1,13 @@
+extern crate clap;
+use self::clap::{Arg, App, SubCommand};
+
+pub fn make() -> App<'static, 'static> {
+    App::new("golden-retriever-indexer")
+        .version("0.0.1")
```

Comment:
Oh, I misunderstood this. This isn't on runtime, but during compile time. That's not bad at all actually.